src/main/java/TaskManager.java

```diff
@@ -0,0 +1,103 @@
+import java.util.ArrayList;
+
+public class TaskManager {
+    ArrayList<Task> tasks;
+
```

Comment:
Good: Plural form used to represent a collection of tasks. This is a common error that you avoided!